protocol/binary/reader_benchmark_test.go

```diff
@@ -0,0 +1,48 @@
+package binary
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"go.uber.org/thriftrw/wire"
+)
+
+func Benchmark_readStruct(b *testing.B) {
+	for _, bench := range []struct {
+		name      string
+		numFields int
+	}{
```

Comment:
stylistic nit: For table tests and benchmarks, please separate the test cases from the body, and place an empty line between them:

```
benchmarks := []struct{
}{
  {
    name: "...",
    numFields: ...,
  },
  {
    // ...
  },
}

for _, bb := range benchmarks {
    b.Run(bb.name, func(b *testing.B) {
        // ...
    })
}
```

Merging the table into the for loops makes it difficult to easily scan where test cases end and where test body begins.